hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,19 @@
+RELEASE_TYPE: minor
+
+This release should make the :class:`~hypothesis.settings` feature
+easier to use.
```

Comment:
I'm not super sold on this line and would rather it be more explicit about what it does. Maybe something like "this release deprecates a number of settings related features, with the goal of making the settings system easier to use"